resources/views/internlink/internship.blade.php

```diff
@@ -0,0 +1,86 @@
+@extends('layouts.panel')
+@section('panel-cols', '10')
+@section('panel-offset', '1')
+@section('panel-heading', 'Internship Profile')
+@section('panel-body')
+    <table id="profile">
+        <tr>
+            <td id="col1">Name</td>
+            <td id="col2">{!! $member_details['name'] !!}</td>
```

Comment:
# Important - blade escaping - security issue
https://laravel.com/docs/5.3/blade#displaying-data

> Blade {{ }} statements are automatically sent through PHP's htmlentities function to prevent XSS attacks.

> Be very careful when echoing content that is supplied by users of your application. Always use the escaped, double curly brace syntax to prevent XSS attacks when displaying user supplied data.

Summary: use `{{ }}` most of the time and only use `{!! !!}` only if you expect the data to be HTML, e.g.  `{!! form($form) !!}`.

I realise that I didn't keep an eye out for this previously. Please look through the previous code (e.g. by using a `git grep` or `ack` command) and remove such instances if possible! My bad, I probably introduced some of this myself...